pages/signup.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="stylesheet" href="https://altur-ui.netlify.app/stylesheets/main.css">
```

Comment:
You can import it directly in the `style.css` file since you are importing `styles.css` in every page. Then, you won't have to import this in every page

Reply:
changed in the latest commit. Thanks for pointing out